sdcm/tester.py

```diff
@@ -1791,6 +1792,10 @@ def run_ycsb_thread(self, stress_cmd, duration=None, stress_num=1, prefix='',
         if self.create_stats:
             self.update_stress_cmd_details(stress_cmd, prefix, stresser="ycsb", aggregate=stats_aggregate_cmds)
 
+        if 'dynamodb.ttlKey' in stress_cmd:
+            node = self.db_cluster.nodes[0]
+            self.alternator.update_table_ttl(node=node, table_name=alternator.consts.TABLE_NAME)
```

Comment:
this is a problematic location, since in some test you might pass here, but actully using a different table.
I think this part should be together with table creation in `pre_create_alternator_tables`
we can add a configuration option to SCT where we would set the configuration of the TTL timeout cycles, we would be able to use that. lookup what the scylla configuration for that and lets add that to ScyllaYaml implementation

Reply:
similar to how we use `alternator_port` which maps to scylla yaml, but also means for us that we are using alternator.

Reply:
fixed